src/main/config/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,57 @@
+{{with $environment := env "ENVIRONMENT"}}{{$keyname := printf "secret/dsde/%s/cromwell/cromwell.conf" $environment}}{{with vault $keyname}}
+
+include "application.conf"
+
+backend {
+  backend = "JES"
+
+  jes {
+    applicationName = "cromwell"
+    project = "{{.Data.jes_project}}"
+    baseExecutionBucket = "{{.Data.jes_base_execution_bucket}}"
+    endpointUrl = "{{.Data.jes_endpoint_url}}"
+    dockerhubCredentialsPath = "{{.Data.jes_dockerhub_credentials_path}}"
```

Comment:
This is has changed too (I'm not even sure it was ever like this).
The way to give cromwell docker credentials is 

```
// Optional Dockerhub Credentials. Can be used to access private docker images.
// Note: Works currently only on JES backend
docker {
  // dockerAccount = ""
  // dockerToken = ""
}```
at the root of the conf file.